draft-ietf-sipcore-sip-push.xml

```diff
@@ -341,6 +341,18 @@ align="center"><artwork>
           notifications towards the UA. The proxy MUST skip the rest of the procedures in this section, and process the
           REGISTER request using normal SIP procedures. 
         </t>
+        <t>
+          If the proxy considers the requested registration expiration interval <xref target="RFC3261"/> to be too short, the
+          proxy MUST either send a 423 (Internal Too Brief) response to the REGISTER request or skip the rest of the procedures 
+          in this section, and process the REGISTER request using normal SIP procedures. Similarly, when the proxy receives a 2xx 
+          response to the REGISTER request (see below), if the proxy considers the registration expiration internal indicated by the registrar too
+          short, the proxy MUST NOT insert a Feature-Caps header field with a 'sip.pns' feature-capability indicator in the response, and the
+          proxy MUST NOT request push notifications associated with the registration.
+        </t>
+        <t>
+         NOTE: A registration expiration interval is considered too short if the interval is smaller than the time prior to expiration 
+         when the proxy requests a push notification.
```

Comment:
I think this could use a little clarification, a proxy should be able to decide if an interval is too short not just based on whether "registration time" > "time before push required". Otherwise in the worst case a UA sends an expiry of n+1 seconds and the proxy has configured time of n seconds. 
What do you think about something like this?
"NOTE: A registration expiration interval MUST be considered too short if the interval is smaller than the time prior to expiration that the proxy would request a push notification. The proxy MAY consider the interval too small based on its own policy so as to reduce load on the system."

Reply:
Works for me. Not sure we can use "MUST" and "MAY" within a NOTE, but I can remove the NOTE part :)